src/makeCalculator.js

```diff
@@ -32,12 +32,44 @@
  *  .operate(calculator.multiply, 7)
  *
  * calculator.result === -28
- *
+
  *
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const calculator = {
```

Comment:
```suggestion
  return {
```
you might immediately return created object instead of creating a separate variable (FYI)